frontend/src/components/fundingAgreements/ManageFundingAgreements.vue

```diff
@@ -4,13 +4,18 @@
       <v-col cols="12">
         <p class="mb-4">View and manage the funding agreements for your organization.</p>
       </v-col>
-      <v-col cols="12" md="9">
+      <v-col cols="8" sm="7">
```

Comment:
I think we can give the paragraph a little more space at small and up. Maybe a 9/3 split?